cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -568,14 +573,15 @@ protected Config brokerLogging(int brokerId) throws ForceableProblem, Interrupte
         );
     }
 
-    protected void dynamicUpdateBrokerConfig(int podId, Admin ac, KafkaBrokerConfigurationDiff configurationDiff, KafkaBrokerLoggingConfigurationDiff logDiff)
+    protected void dynamicUpdateBrokerConfig(PodRef podRef, Admin ac, KafkaBrokerConfigurationDiff configurationDiff, KafkaBrokerLoggingConfigurationDiff logDiff)
             throws ForceableProblem, InterruptedException {
         Map<ConfigResource, Collection<AlterConfigOp>> updatedConfig = new HashMap<>(2);
+        var podId = podRef.podId;
         updatedConfig.put(Util.getBrokersConfig(podId), configurationDiff.getConfigDiff());
         updatedConfig.put(Util.getBrokersLogging(podId), logDiff.getLoggingDiff());
 
-        LOGGER.debugCr(reconciliation, "Altering broker configuration {}", podId);
-        LOGGER.traceCr(reconciliation, "Altering broker configuration {} with {}", podId, updatedConfig);
+        LOGGER.debugCr(reconciliation, "Altering broker configuration {}", podRef);
```

Comment:
Out of curiosity ... how does the `PodRef` translate into the log? It seems to have its own `toString` method with which seems to use for whatever reason `"{" + podName + '}'`

Reply:
Yeah it does translate into `{my-cluster-0}` which is a bit of a weird notation.

Reply:
Done